ci/setup-build-test-gdk.ps1

```diff
@@ -65,7 +66,7 @@ class TestSuite {
 [string] $test_map_path = "/Game/Intermediate/Maps/"
 
 [TestProjectTarget] $gdk_test_project = [TestProjectTarget]::new("git@github.com:spatialos/UnrealGDKTestGyms.git", $gdk_branch, "Game\GDKTestGyms.uproject", "GDKTestGyms", $gyms_version_path, "env:TEST_REPO_BRANCH")
-[TestProjectTarget] $native_test_project = [TestProjectTarget]::new("git@github.com:improbable/UnrealGDKEngineNetTest.git", $gdk_branch, "Game\EngineNetTest.uproject", "NativeNetworkTestProject", $gyms_version_path, "env:NATIVE_TEST_REPO_BRANCH")
+[TestProjectTarget] $native_test_project = [TestProjectTarget]::new("git@github.com:improbable/UnrealGDKEngineNetTest.git", $gdk_branch, "Game\EngineNetTest.uproject", "NativeNetworkTestProject", "", "env:NATIVE_TEST_REPO_BRANCH")
```

Comment:
Actually, I think I would rather keep the gyms_version_path
Otherwise this could cause issues during release cycles.
It's not the most elegant thing to make the testgym version file be used for this as well, but it's the least code delta to avoid failing builds for future rcs.

Reply:
This was not fun to deal with when testing the 4.26.1 merge and was causing issues because this branch did not exist on UnrealGDKEngineNetTest. I think its pretty bad to keep it as is.

Reply:
gyms_version_path having effects on native test build is a pretty nasty and unintuitive side effect.

Reply:
I think I'll add another text file instead

Reply:
You would have to change the release pipeline to account for it as well

Reply:
Hmm, the 0.12.0 tag should exist for the engine net test: https://github.com/improbable/UnrealGDKEngineNetTest/releases/tag/0.12.0